backend/src/main/java/com/bakdata/conquery/models/query/queryplan/specific/FiltersNode.java

```diff
@@ -36,7 +37,7 @@ private FiltersNode(List<? extends FilterNode<?>> filters, List<EventFilterNode<
 		// This links them up as a back-reference.
 		for (Aggregator<?> aggregator : aggregators) {
 			if (aggregator instanceof ExistsAggregator) {
-				((ExistsAggregator) aggregator).setFilters(this);
+				((ExistsAggregator) aggregator).setParent(this);
 			}
 		}
 	}
```

Comment:
Das Sortieren der Filter gehört für mich in den Kontruktor und macht dann die `create` Methode überflüssig.

Reply:
ja verstehe ich, aber ich versuche logik aus konstruktoren zu halten. aber kanns auch so machen.